src/aditiGJUN20/PrimeNumberOps.java

```diff
@@ -0,0 +1,62 @@
+/*
+Create a class to satisfy below requirements. 
+	a) Given number is prime or not. 
+	b) Print total count of prime numbers between 2->100.
+    c) Find sum of all prime numbers between 2->100
+    d) Find avg of prime numbers between 2->100
+ */
+
+package aditiGJUN20;
+
+public class PrimeNumberOps {
+	int primeNumAddition=0, primeCnt=0;
+	
+	// Method contains logic to number is prime or not
+	boolean isPrime(int number) {
+		boolean flag=true;	
+		for(int index=2;index<=Math.sqrt(number);index++) {
+			if(number % index == 0) {
+				flag=false;
+				break;
+			}
+		}	
+		return flag;
+	}
+	
+	// Using isPrime() Method, displaying number is prime or not
+	void getPrimeNum(int num) {
+		boolean flag=isPrime(num);
+		if(flag==true)
+			System.out.println(num+" is Prime Number.");
+		else 
+			System.out.println(num+" is not Prime Number.");
+	}
+	
+	// Method to find prime numbers in given range.
+	void arithmaticOpsOnPrimeNum() {
+		boolean flag=false;
+		System.out.println("\nPrime numbers within 2-100 range: ");
```

Comment:
No need to write extra methods, Just create two instance variable sum and count and perform that operation in isPrime method and use those variable in method like:
void findPrimeNumberInGivenRange(int startNumber, int endNumber) {
for(int index=startNumber; index<=endNumber; index++) {
isPrime(index);
}
System.out.println("Total count is: "+count);
System.out.println("Sum of all Prime number is: "+sum);
System.out.println("Average of all prime number is: "+sum/count);